docs/quickstart/quickstart_chains/base.md

```diff
@@ -0,0 +1,348 @@
+# Base (Goerli) Quick Start
+
+## Goals
+
+The goal of this quick start guide is to index the total faucets dripped to users from the [USDC Faucet contract](https://goerli.basescan.org/address/0x298e0b0a38ff8b99bf1a3b697b0efb2195cfe47d) on [Base Goerli Testnet](https://docs.base.org/using-base/).
+
+::: warning
+Before we begin, **make sure that you have initialised your project** using the provided steps in the [Start Here](../quickstart.md) section. Please initialise an a Base project.
+:::
+
+In every SubQuery project, there are [3 key files](../quickstart.md#_3-make-changes-to-your-project) to update. Let's begin updating them one by one.
+
+::: tip Note
+The final code of this project can be found [here](https://github.com/subquery/ethereum-subql-starter.git). We use Ethereum packages, runtimes, and handlers (e.g. @subql/node-ethereum, ethereum/Runtime, and ethereum/*Handler) for Arbitrum. Since Arbitrum is an EVM-compatible layer-2 scaling solution, we can use the core Ethereum framework to index it.
```

Comment:
```suggestion
The final code of this project can be found [here](https://github.com/subquery/subquery-example-base-faucet). 

We use Ethereum packages, runtimes, and handlers (e.g. @subql/node-ethereum, ethereum/Runtime, and ethereum/*Handler) for Arbitrum. Since Arbitrum is an EVM-compatible layer-2 scaling solution, we can use the core Ethereum framework to index it.
```